server/aap/commands/export_legal_archive_to_archive.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014, 2015, 2016, 2017 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+import superdesk
+import json
+
+from superdesk.celery_task_utils import get_lock_id
+from superdesk.lock import lock, unlock
+from superdesk import get_resource_service
+from eve.utils import ParsedRequest
+from apps.legal_archive import LEGAL_ARCHIVE_NAME
+
+from copy import deepcopy
+
+logger = logging.getLogger(__name__)
+
+
+class ExportLegalArchiveToArchive(superdesk.Command):
+    """Export legal archive content to archive.
+
+    Only the content for the desk provided is exported.
+    By default this is the 'COMMISSION' desk
+    """
+
+    default_desk = 'COMMISSION'
+
+    option_list = [
+        superdesk.Option('--desk', '-d', dest='desk', required=False)
+    ]
+
+    def run(self, desk=None):
+        if desk:
+            self.default_desk = desk
+
+        logger.info('Starting to export {} desk legal archive content to archive'.format(self.default_desk))
+
+        lock_name = get_lock_id('legal_archive', 'export_to_archive')
+        if not lock(lock_name, expire=610):
+            logger.info('Export legal archive to archive task is already running.')
+            return
+        try:
+            self._export_to_archive()
```

Comment:
`_export_to_archive` can return None if desk is not found so it may be better to log another message like `Completed bot nothing exported..`